spec.example.yaml

```diff
@@ -24,12 +24,37 @@ prometheus:
 workers:
   count: 6
   machineType: c4.large
-  appInstances: 180
-loadGenerator:
```

Comment:
@fabxc Why are we configuring the *load generator* in the "workers" section. I would prefer if we keep the "loadGenerator" section. The *querier* configuration e.g. has nothing to do with the *workers*, right?